orchestra/signals.py

```diff
@@ -0,0 +1,11 @@
+from django.dispatch import receiver
+from registration.signals import user_registered
+
+from orchestra.models import Worker
+
+
+@receiver(user_registered)
+def add_worker_for_new_users(sender, user, request, **kwargs):
+    # TODO(dhaas): Figure out how to get the registration app to pass in the
```

Comment:
maybe just make an issue for it, and don't mention slack or `slack_username` in this file